src/test/java/com/redhat/devtools/lsp4ij/operations/documentation/LSPDocumentationProviderTest.java

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Red Hat Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *
+ * Contributors:
+ *     Red Hat Inc. - initial API and implementation
+ *******************************************************************************/
+package com.redhat.devtools.lsp4ij.operations.documentation;
+
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class LSPDocumentationProviderTest {
+
+    private LSPDocumentationProvider docProvider;
+
+    @Before
+    public void setup(){
+        docProvider = new LSPDocumentationProvider();
+    }
+
+    @Test
+    public void markdownTableToHTML() {
+        String markdown = """
+                Does something, `somewhere`:
+                
+                |                    |                    |
+                |--------------------|--------------------|
+                | Row One - Cell One | Row One - Cell Two |
+                | Row Two - Cell One | Row Two - Cell Two |
+                """;
+
+        String expected = """
+                <p>Does something, <code>somewhere</code>:</p>
+                <table>
+                <thead>
+                <tr><th> </th><th> </th></tr>
+                </thead>
+                <tbody>
+                <tr><td>Row One - Cell One</td><td>Row One - Cell Two</td></tr>
+                <tr><td>Row Two - Cell One</td><td>Row Two - Cell Two</td></tr>
+                </tbody>
+                </table>
+                """;
+        assertEquals(expected, docProvider.toHTML(markdown));
+    }
```

Comment:
Could you please write other tests with other html elements like <p>, <b>, <i>, etc like we have https://github.com/eclipse/lemminx/blob/main/org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/utils/MarkdownConverterTest.java (but perhaps by splitti,g them in several tests)

Could you write too a simple text with hyperlink. I have not a usecase in my mind, but perhaps we should support baseURI in the future, If we have one test with hyperlink we will able to adapt it with baseURI

Do you know if the converter could support github syntax for code (ex {'''xml)), but I think we need to integrate that with IJ and textmate (I did that in eclipse tm4e markdown support).

Reply:
done